.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
               command: |
                   export PATH=~/miniconda/bin:$PATH
                   conda update --yes --quiet conda
-                  conda create -n testenv --yes pip python=3.7
+                  conda create -n testenv --yes pip python=3.8
```

Comment:
Do you think we need to update `setup.py` and our unit tests with `python>=3.8` as well to ensure all builds are getting tested properly?

Reply:
that's true but you won't realize if other dependencies start requiring 3.8 now that you have updated it here. I don't think we have a CI that tests for python < 3.8? Better to be consistent and update readme/setup etc.

Reply:
I went ahead and updated this in #677. We'll see if it fixes the Windows `test_viz.py` error....

Reply:
excellent!!